_posts/2018-02-27-duplication-you-are-welcome.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+asset-type: post
+name: duplication-you-are-welcome 
+title: Duplication, you're welcome
+description: Thoughts and misunderstandings about duplication
+date: 2018-02-27 18:00:00 +00:00
+author: Raquel M Carmena
+image:
+   src: /assets/custom/img/blog/2018-02-27-duplication-you-are-welcome/duplication-header.jpg
+tags:
+- quality
+- duplication
+- metrics
+
+---
+
+There are models to describe the process of learning as <a href="http://wiki.c2.com/?DreyfusModel">Dreyfus model</a> or <a href="http://wiki.c2.com/?ShuHaRi" target="_blank">Shuhari</a>, but I'd explain another process when learning a new technical skill. It would be composed by 2 phases: 
+
+1. **Fascination**. When we learn a new concept that sounds really good to solve our problems and we apply it in any place we have an apportuninity. And we are so convinced about its advantages that it seems we have a radar to detect a place in which we can apply it. It's like the first stage at Shuhari, but we are absolutely fascinated and we don't believe in _another world_.
+
+2. **Questioning**. When we have assimilated that concept and we are able to decide where it's convenient to implement it. We have learnt from the previous stage and now we are moderate and reasonable about our decisions.
```

Comment:
Again, begin with "This is when..." or you could use "Questioning is when..." for variety if you like.